src/control/settings/ButtonConfig.cpp

```diff
@@ -28,18 +28,18 @@ void ButtonConfig::initButton(ToolHandler* toolHandler, Button button) {
     toolHandler->resetButtonTool(this->action, button);
 
     if (this->action == TOOL_PEN || this->action == TOOL_HIGHLIGHTER || this->action == TOOL_ERASER) {
```

Comment:
@rolandlo : This part I only changed to make the flow similar to `ButtonConfig::applyNoChangeSettings`, which was the function that actually needed to change.
So in this function the changes should be logically equivalent to the version before.